yaksh/views.py

```diff
@@ -176,9 +185,24 @@ def edit_question(request, question_id=None):
         raise Http404('No Question Found')
 
     question_instance = Question.objects.get(id=question_id)
-
+    if request.method == "POST" and 'delete_files' in request.POST:
+        remove_files_id = request.POST.getlist('clear')
+        if remove_files_id:
+            files = QuestionsFileUpload.objects.filter(id__in=remove_files_id)
+            file = QuestionsFileUpload()
+            file.delete_files(files)
```

Comment:
In models you have created delete_files, so that you can use file.delete() where file is the object instance but here you are creating new file instance and then calling file.delete() with argument again of same instance type.
I think your delete method should be in File Manager in models. So that you can simply do File.objects.delete(files)
Note: Also django queryset provides delete method.